### ISSUE TITLE
ContainerAnvil m() throws NullPointerException

### ISSUE THREAD

**Issue by FloGamerMC** [pl3xgaming/Purpur#1344]
### Spark link

don't have it

### Expected behavior

I don't know what this method do, because it's a [library](https://github.com/WesJD/AnvilGUI/blob/master/1_20_R1/src/main/java/net.wesjd.anvilgui.version/Wrapper1_20_R1.java#L96). But I tested it with the newest Paper version and it worked. Maybe you must add the newst paper patches. 

### Observed/Actual behavior

The method throws an exception:
```
java.lang.NullPointerException: Cannot invoke "String.length()" because "input" is null
	at net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializerImpl.deserialize(LegacyComponentSerializerImpl.java:207) ~[adventure-text-serializer-legacy-4.14.0.jar:?]
	at net.minecraft.world.inventory.AnvilMenu.createResult(AnvilMenu.java:342) ~[?:?]
	at net.wesjd.anvilgui.version.Wrapper1_20_R1$AnvilContainer.m(Wrapper1_20_R1.java:96) ~[Fuxelhunt.jar:?]
	at net.minecraft.world.inventory.ItemCombinerMenu.slotsChanged(ItemCombinerMenu.java:112) ~[?:?]
	at net.minecraft.world.inventory.ItemCombinerMenu$3.setChanged(ItemCombinerMenu.java:103) ~[?:?]
	at net.minecraft.world.SimpleContainer.setItem(SimpleContainer.java:207) ~[?:?]
	at org.bukkit.craftbukkit.v1_20_R1.inventory.CraftResultInventory.setItem(CraftResultInventory.java:37) ~[purpur-1.20.jar:git-Purpur-1990]
	at net.wesjd.anvilgui.AnvilGUI.openInventory(AnvilGUI.java:170) ~[Fuxelhunt.jar:?]
	at net.wesjd.anvilgui.AnvilGUI.access$1000(AnvilGUI.java:37) ~[Fuxelhunt.jar:?]
	at net.wesjd.anvilgui.AnvilGUI$Builder.open(AnvilGUI.java:561) ~[Fuxelhunt.jar:?]

```

### Steps/models to reproduce

I used this library AnvilGUI, on the Point I want to open the anvil gui this exception came up. But as mentioned above with the newest paper build it works fine.

### Purpur version

[19:12:34 INFO]: Checking version, please wait...
[19:12:34 INFO]: Current: git-Purpur-1990 (MC: 1.20)*
Previous: git-Paper-10 (MC: 1.20)
* You are running the latest version

### Agreements

- [X] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [ ] I ticked all the boxes without actually reading them
- [X] My version of Minecraft is supported by Purpur.

**Comment by Gypopo** [comment]
Confirmed on latest 1.20.1 https://hastebin.com/share/gorobusoro.ruby

**Comment by shawn695** [comment]
i am also having this issue but with economyshopgui premium 
works on purpur 1.19.4 and im running purpur 1.20

14.06 14:47:25 [Server] INFO Current: git-Purpur-1990 (MC: 1.20)*
14.06 14:47:25 [Server] INFO Previous: git-Purpur-1976 (MC: 1.19.4)
14.06 14:47:25 [Server] INFO * You are running the latest version
[message.txt](https://github.com/PurpurMC/Purpur/files/11747322/message.txt)

**Comment by FloGamerMC** [comment]
I already fixed the bug my self, we just need to wait until the pull request get accepted.

**Comment by granny** [comment]
Should be fixed as of build #1993. Let me know if that's the case.
